Client-Server/Server/SeTCPServer.m

```diff
@@ -0,0 +1,88 @@
+#import "SeTCPServer.h"
+#import <Foundation/Foundation.h>
+#import <CoreFoundation/CoreFoundation.h>
+#import <sys/socket.h>
+#import <netinet/in.h>
+
+@interface SeTCPServer()
+
+@property (nonatomic) CFSocketRef socket;
+@property (nonatomic) NSInputStream *inputStream;
+
+@end
+
+@implementation SeTCPServer
+
+- (id)init {
+    return [self initWithPort:6666];
+}
+
+- (id)initWithPort:(NSUInteger)port {
+    if(([super init])) {
+        _port = port;
+    }
+
+    return self;
+}
+
+- (void)start {
+    self.socket = CFSocketCreate(kCFAllocatorDefault, PF_INET, SOCK_STREAM, IPPROTO_TCP, kCFSocketAcceptCallBack, serverConnectCallback, nil);
+   
+    if(!self.socket) {
+        NSLog(@"CFSocketCreate failed");
+        return;
+    }
+
+    struct sockaddr_in sin = {0};
```

Comment:
rename to something more indicative, not everybody knows sin==socket inet address :P